app/src/main/java/net/aiscope/gdd_app/ui/mask/customview/MaskCustomView.kt

```diff
@@ -57,13 +61,13 @@ class MaskCustomView @JvmOverloads constructor(
     private fun onTouchMove(event: MotionEvent) = attacher.onTouch(this, event)
 
     private fun onTouchDraw(event: MotionEvent): Boolean {
-        val (x, y) = invert(event.x, event.y)
-        val (drawableWidth, drawableHeight) = drawableDimensions
         if (event.action == MotionEvent.ACTION_UP) {
             maskLayer.drawEnd()
             onMaskingActionFinishedListener?.onTouch(this, event)
             invalidate()
         } else {
+            val (x, y) = invert(event.x, event.y)
+            val (drawableWidth, drawableHeight) = drawableSize
```

Comment:
I also moved these closer to where they are used.

Reply:
Nice! Thanks!!